content/en/tracing/setup_overview/proxy_setup/_index.md

```diff
@@ -53,66 +53,66 @@ Datadog APM is included in Envoy v1.9.0 and newer.
 
 ## Enabling Datadog APM
 
-**Note**: The example configuration below is for Envoy v1.14.
-Example configurations for older versions can be found [here][1]
+**Note**: The example configuration below is for Envoy v1.19.
+Example configurations for other versions can be found [in the `dd-opentracing-cpp` GitHub repo][1].
 
-Three settings are required to enable Datadog APM in Envoy:
+The following settings are required to enable Datadog APM in Envoy:
 
 - a cluster for submitting traces to the Datadog Agent
 - `tracing` configuration to enable the Datadog APM extension
 - `http_connection_manager` configuration to activate tracing
 
-A cluster for submitting traces to the Datadog Agent needs to be added.
-
-```yaml
-  clusters:
-  ... existing cluster configs ...
-  - name: datadog_agent
-    connect_timeout: 1s
-    type: strict_dns
-    lb_policy: round_robin
-    load_assignment:
-      cluster_name: datadog_agent
-      endpoints:
-      - lb_endpoints:
-        - endpoint:
-            address:
-              socket_address:
-                address: localhost
-                port_value: 8126
-```
-
-The `address` value may need to be changed if Envoy is running in a container or orchestrated environment.
-
-Envoy's tracing configuration needs to use the Datadog APM extension.
-
-```yaml
-tracing:
-  http:
-    name: envoy.tracers.datadog
-    typed_config:
-      "@type": type.googleapis.com/envoy.config.trace.v2.DatadogConfig
-      collector_cluster: datadog_agent   # matched against the named cluster
-      service_name: envoy-example        # user-defined service name
-```
+1. Add a cluster for submitting traces to the Datadog Agent:
+
+   ```yaml
+    clusters:
+    ... existing cluster configs ...
+    - name: datadog_agent
+      connect_timeout: 1s
+      type: strict_dns
+      lb_policy: round_robin
+      load_assignment:
+        cluster_name: datadog_agent
+        endpoints:
+        - lb_endpoints:
+          - endpoint:
+              address:
+                socket_address:
+                  address: dd-agent
```

Comment:
The address should probably be `localhost`. An address like `dd-agent` only works in a specific environment, such as a docker-compose setup with a container named `dd-agent`.